test/basic.js

```diff
@@ -15,6 +15,11 @@ describe('Basic', () => {
     }, 1000);
   });
 
+  it('should only test in the correct environment', function(done) {
+      this.skip();
```

Comment:
if you call `skip()` on `it` instance, you'll be able to use arrow function here.
```js
it.skip(`this test`, () => {
...
});
```

Reply:
Yes, but sometimes we need to skip at runtime.
Like the following code,
```
  if (/* check test environment */) {
    // make assertions
  } else {
    this.skip();
  }
```

So I am not using arrow function here.